package.json

```diff
@@ -1,16 +1,17 @@
 {
   "name": "webpack-build-notifier",
-  "version": "2.3.0",
+  "version": "3.0.0",
   "description": "A Webpack plugin that generates OS notifications for build steps using node-notifier.",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "type": "commonjs",
```

Comment:
`npm test` is failing since the package type is now `commonjs` but `jest.config.js` file is an ES module. 🙂 

Do we need to specify `commonjs` here?